docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -119,6 +119,74 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Receiving events
```

Comment:
I'm wondering what would be better here – putting this section before the API section or after? I feel like, as a docs reader, I would like to read the API first and then some guides, more advanced examples etc.

Reply:
I feel like in most libraries we keep the usage examples at the top, so I guess the current way would be more consistent